### ISSUE TITLE
Enable webp will prevent image cached in result_storage

### ISSUE THREAD

**Issue by augustine-tran** [APSL/docker-thumbor#54]
I noticed that after enable AUTO_WEBP=True then image don't auto save to result_storage folder. Here is my docker-compose.yml config
```
thumbor:
    image: apsl/thumbor
    volumes:
      - ./stack/thumbor/.data:/data
    environment:
      - ALLOW_UNSAFE_URL=True
      - SECURITY_KEY=Cyberdyne2017
      - RESULT_STORAGE_STORES_UNSAFE=True
      - AUTO_WEBP=True
  nginx:
    image: apsl/thumbor-nginx:latest
    links:
      - thumbor:thumbor
    volumes:
      - ./stack/thumbor/.data:/data
    ports:
      - 12003:80
```

Not sure if its related to this issue [Webp from result_storage is not setting Vary header #731
](https://github.com/thumbor/thumbor/issues/731).

**Comment by eduherraiz** [comment]
Thanks for report the issue @rualatngua .
I found the webp images are stored in a different way, using a subfolder called "webp" inside the result_storage. Because of that, the nginx can't found them.
Let me think a while how to deal with this.

**Comment by eduherraiz** [comment]
We will have to implement a webp use case in nginx:
https://github.com/uhop/grunt-tight-sprite/wiki/Recipe:-serve-WebP-with-nginx-conditionally

**Comment by eduherraiz** [comment]
Implemented a way to do the webp cache in nginx in the branch 6.2.0b.
https://github.com/APSL/docker-thumbor/compare/6.2.0b

Now, thumbor-nginx is smart enough to know if the browser allows webp, and use the correct directory cache (webp or not) or do the failover to thumbor.

@rualatngua You can found the tag 6.2.0b in the docker hub to test it!
I will test it a few days before release to master/latest. 

You need to include the AUTO_WEBP=True envvar to thumbor-nginx too!

**Comment by eduherraiz** [comment]
:clap: :clap: :clap:
A public applause to @paurullan that helps to debug a very tricky nginx config problem: https://trac.nginx.org/nginx/ticket/1142#ticket

**Comment by augustine-tran** [comment]
Thank @eduherraiz. Let me try 6.2.2b in a few days and back to you soon.

**Comment by eduherraiz** [comment]
Available in 6.2.0b version
